weblab_cg/_generator_chaste_cg.py

```diff
@@ -50,15 +51,14 @@ class ChasteModel(object):
                    'membrane_stimulus_current_end': [{'units': 'millisecond'}],
                    'membrane_capacitance': [{'units': 'uF'}, {'units': 'uF_per_mm2'}]}
 
-    _STIM_RHS_MULTIPLIER = {'membrane_stimulus_current_amplitude':
-                            {'uA_per_uF': sp.Function(_HEARTCONFIG_GETCAPACITANCE)(),
-                             'uA': sp.Function(_HEARTCONFIG_GETCAPACITANCE)()}}
-
     _STIM_CONVERSION_RULES_ERR_CHECK = \
```

Comment:
Might be useful to have some comments explaining how these are used?

Reply:
Ok good point